app/src/main/java/com/example/plantsapp/data/firebase/repository/FirebasePlantsRepository.kt

```diff
@@ -30,10 +42,16 @@ class FirebasePlantsRepository @Inject constructor(
                         }
 
                         value != null -> {
-                            val plants = value
-                                .toObjects<FirebasePlant>()
-                                .map { it.toPlant() }
-                            trySend(plants)
+                            launch {
```

Comment:
Why you started one more launch?

Reply:
needed it here for accessing suspend function to save image locally